config/kibana.yml

```diff
@@ -1,8 +1,11 @@
 # Kibana is served by a back end server. This setting specifies the port to use.
 # server.port: 5601
 
-# This setting specifies the IP address of the back end server.
-# server.host: "0.0.0.0"
+# Specifies the host name to which the Kibana server will bind.
+# The default is 'localhost', which usually means remote machines will not be able to connect.
+# To allow connections from remote users, set this parameter to IP or the DNS-name of the machine that is running Kibana.
+# For example: if remote users need to connect to Kibana via http://foobar:5601, set this parameter to 'foobar'.
+# server.host: "localhost"
```

Comment:
I'd hesitate for that.

Right now, we're borrowing the semantics and implementation straight from `hapi`. If we uncouple "public name"(=host) and "IP assigned to network interface"(=listen), we'll have to implement this ourselves.

Has this been a problem in the past?